### ISSUE TITLE
Bug: Unable to undo changes in commit message

### ISSUE THREAD

**Issue by emmadesilva** [desktop/desktop#17822]
### The problem

This issue is becoming increasingly infuriating. For example, just now, I wrote a detailed commit description, then remembered I needed to amend, so I do that -- but now the commit message box is cleared. No worries I think, intuition tells me that I can hit <kbd>CTRL+Z</kbd> to recover it, but of course that doesn't work!

This has also happened on occasion after removing text from the inputs, with no amend involved.

TL;DR: Please support "undos" in the commit summary and description boxes to recover changes!

### Release version

3.3.5

### Operating system

Windows 10

### Steps to reproduce the behavior

1. Write something in the summary/description boxes
2. Go to history and amend a commit, the current summary is cleared
3. Try to undo the changes in the summary, nothing happens after a CTRL+Z

**Comment by steveward** [comment]
Thanks for the issue @caendesilva! I'll mark this as an enhancement for future consideration.
